### ISSUE TITLE
Null safety required

### ISSUE THREAD

**Issue by jttuboi** [acaziasoftcom/flutter_kanji_view#2]
Hi, I like your project and I want to use in my project. But it's not null safety. Is it any chance to update it?

I looked your LICENSE, but it doesn't have license. Is it possible to copy your project in case you can't update?

**Comment by Doko-Demo-Doa** [comment]
Yes, it's planned. Actually I was working on a local, separated branch several weeks ago, but paused due to health issue. I think I will be back soon.

**Comment by jttuboi** [comment]
Thanks. I'll wait for it. I hope you get well soon.
